### PR TITLE
feat(dashboard): date and time on tooltip

### LINKED ISSUE #228: date columns: add time in a hover
Add hover behavior to the Tree Listing Page:
Screenshot 2024-09-12 at 16 53 45

### PR DESCRIPTION
**Pull request by lucasbracher**
Date and time now are shown when hovering over date.

| Before | After |
|--------|--------|
| image | image |

Closes #228

### CODE REVIEW

File: dashboard/src/components/Table/TreeTable.tsx [hunk 1]
```diff
@@ -134,7 +137,18 @@ const TreeTableRow = (row: TreeTableBody): JSX.Element => {
         onClick={navigateToTreeDetailPage}
         data-target="treeDetails.builds"
       >
-        {sanitizeTableValue(row.date.split('T')[0] ?? '')}
+        <Tooltip>
+          <TooltipTrigger>
+            <div>{date}</div>
+          </TooltipTrigger>
+          <TooltipContent>
+            <div style={{ textAlign: 'center' }}>
```
Comment (WilsonNet) [inline dashboard/src/components/Table/TreeTable.tsx:145]: use tailwind class

https://tailwindcss.com/docs/text-align

**Review by WilsonNet — COMMENTED** [review]
It is working fine, just one minor comment, and please leave a screenshot on your PR of the feature

**Review by WilsonNet — APPROVED** [review]
nice

File: dashboard/src/components/Table/TreeTable.tsx [hunk 2]
```diff
@@ -101,6 +101,9 @@ const TreeTableRow = (row: TreeTableBody): JSX.Element => {
     ],
   );
 
+  const date = row.date.split('T')[0] ?? '';
+  const time = row.date.split('T')[1].split('.')[0] ?? '';
```
Comment (lfjnascimento) [inline dashboard/src/components/Table/TreeTable.tsx:105]: Will be safer if we use the `Date` obj
```
  const dateObj = new Date(row.date);
  const date = dateObj.toLocaleDateString();
  const time = dateObj.toLocaleTimeString();
```

**Pull request merged**